pkg/utils/util.go

```diff
@@ -267,12 +277,16 @@ func getSecretReference(snapshotClassParams map[string]string, snapContentName s
 	}
 	ref.Namespace = resolvedNamespace
 
-	// Secret name template can make use of the VolumeSnapshotContent name, VolumeSnapshot name or namespace.
+	// Secret name template can make use of the VolumeSnapshotContent name, VolumeSnapshot name or namespace,
+	// or a VolumeSnapshot annotation.
```

Comment:
We removed support for annotations right?

Reply:
Yes.  Bad merge.  Removed this now.

Reply:
also update comment

Reply:
done